### PR TITLE
Refactor the geometries section

### PR DESCRIPTION
**Pull request by fabienrohrer**
Before: https://www.cyberbotics.com/doc/guide/geometries
After: https://www.cyberbotics.com/doc/guide/geometries?version=enhancement-geometries-section

### CODE REVIEW

**Comment by fabienrohrer** [comment]
Note: I grouped `box.wbt`, `capsule.wbt`, `cone.wbt`, `convex_polygon.wbt`, `cylinder.wbt`, `polyhedra.wbt`, `sphere.wbt` in a single section without screenshot because they are in a bad state. I want to fix them in R2018a.revision1: https://github.com/omichel/webots/issues/6765

**Comment by fabienrohrer** [comment]
@tn12787 Could you check english in this PR?

**Comment by tn12787** [comment]
done and commited 💯

**Comment by fabienrohrer** [comment]
👍

**Comment by tn12787** [comment]
Nice work!

File: guide/geometries.md [hunk 1]
```diff
@@ -5,14 +5,97 @@ The world files for these examples are located in the "sample/geometries/worlds"
...
+- `cylinder.wbt`: demonstrates how to use the `Cylinder` node to display a red cylinder.
+- `polyhedra.wbt`: demonstrates how to use the `IndexedFaceSet` node to display a red polyhedra.
+- `sphere.wbt`: demonstrates how to use the `Sphere` node to display a red sphere.
+
+### extended\_solids.wbt
+
+**Keywords**: Extended solids, torus, rounded box, pipe
+
+![extended_solids.png](images/extended_solids.png) This example demonstrates the use of several extended primitives.
+These compound primitives are created using procedural PROTOs that generate a set of basic Geometry nodes.
```
Comment (stefaniapedrazzi) [inline guide/geometries.md]: We do not use PROTOs in the documentation. Instead we write PROTO nodes, PROTO files, etc.

**Comment by fabienrohrer** [comment]
@stefaniapedrazzi Thank you I forgot this rule. Is it ok like this?

👍

**Comment by fabienrohrer** [comment]
👍

**Pull request merged**